tools/sense_studio/static/main.js

```diff
@@ -145,10 +145,10 @@ $.fn.form.settings.rules.uniqueClassName = function (className) {
 }
 
 
-function syncRequest(url, data) {
+function syncRequest(url, data, method='POST') {
     let xhttp = new XMLHttpRequest();
 
-    xhttp.open('POST', url, false);
+    xhttp.open(method, url, false);
     xhttp.setRequestHeader('Content-type', 'application/json; charset=utf-8');
```

Comment:
`GET` request do not need RequestHeader, so here you can use method variable as condition.